src/Controls/src/Core/PlatformConfiguration/WindowsSpecific/Page.cs

```diff
@@ -49,30 +49,49 @@ public static ToolbarPlacement GetToolbarPlacement(this IPlatformElementConfigur
 
 		#region ToolbarDynamicOverflowEnabled
 
-		/// <summary>Bindable property for attached property <c>ToolbarDynamicOverflowEnabled</c>.</summary>
+		/// <summary>
+		/// Indicates whether toolbar items automatically move to the overflow menu when space is limited.
+		/// </summary>
 		public static readonly BindableProperty ToolbarDynamicOverflowEnabledProperty =
 			BindableProperty.CreateAttached("ToolbarDynamicOverflowEnabled", typeof(bool),
 				typeof(FormsElement), true);
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific/Page.xml" path="//Member[@MemberName='GetToolbarDynamicOverflowEnabled'][1]/Docs/*" />
+		/// <summary>
+		/// Gets a value that indicates whether toolbar items automatically move to the overflow menu when space is limited.
+		/// </summary>
+		/// <param name="element">A page, the VisualElement that occupies the entire screen.</param>
+		/// <returns><see langword="true"/> if toolbar items automatically move to the overflow menu when space is limited; otherwise, <see langword="false"/>.</returns>
 		public static bool GetToolbarDynamicOverflowEnabled(BindableObject element)
 		{
 			return (bool)element.GetValue(ToolbarDynamicOverflowEnabledProperty);
 		}
 
-		/// <include file="../../../../docs/Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific/Page.xml" path="//Member[@MemberName='SetToolbarDynamicOverflowEnabled'][1]/Docs/*" />
+		/// <summary>
+		/// Sets a value that indicates whether toolbar items automatically move to the overflow menu when space is limited.
+		/// </summary>
+		/// <param name="element">A page, the VisualElement that occupies the entire screen.</param>
+		/// <param name="value">A value that indicates whether toolbar items automatically move to the overflow menu when space is limited</param>
```

Comment:
I recommend adhering to style guides for boolean parameters: https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors